Sources/SwiftTwitch/Classes/Results/Users/UserFollowData.swift

```diff
@@ -30,16 +30,4 @@ public struct UserFollowData: Unmarshaling {
     /// relationship.
     public let followedUserName: String
 
-    /// Initializes a `UserFollowData` object from the input `MarshaledObject`. This will throw if
-    /// there is missing data from the input `MarshaledObject`.
-    ///
-    /// - Parameter object: The object to initialize a `UserFollowData` object from
-    /// - Throws: If data is missing that was expected to be non-`nil`.
-    public init(object: MarshaledObject) throws {
-        followerId = try object.value(for: Twitch.WebRequestKeys.fromId)
-        followerName = try object.value(for: Twitch.WebRequestKeys.fromName)
-        followedUserId = try object.value(for: Twitch.WebRequestKeys.toId)
-        followedUserName = try object.value(for: Twitch.WebRequestKeys.toName)
```

Comment:
Different property name vs web request key name, need to change variable name or do custom Codable initialization